assignment/impl/src/java/org/sakaiproject/assignment/impl/EmailUtil.java

```diff
@@ -190,8 +193,10 @@ public Map<String, Object> getReleaseResubmissionReplacements(AssignmentSubmissi
         }
 
         String linkToToolInSite = "<a href=\"" + getAssignmentUrl(assignment) + "\">" + replacements.get("siteTitle") + "</a>";
-        replacements.put("assignmentUrl", linkToToolInSite);
-        replacements.put("assignmentTitle", assignment.getTitle());
+        replacements.put("url", linkToToolInSite);
+        addIcon(replacements);
+        replacements.put("title", assignment.getTitle());
+        addIcon(replacements);
```

Comment:
Does this `addIcon `need to execute twice? Not sure if I'm missing something here.